### ISSUE TITLE
Configure travis to deploy instructions for a package manager

### ISSUE THREAD

**Issue by Kreyren** [Winetricks/winetricks#1443]
TLDR: Configure travis to generate downstream for apt.

Based on https://github.com/Winetricks/winetricks/issues/1192, https://github.com/Winetricks/winetricks/issues/1257 + similar and previous experience with ubuntu i'm proposing adapting Travis-CI to deploy downstream for winetricks assuming that we already have the logic for pbuilder or simmilar method of packaging `.deb` which would just update the version and packaging.

Referencing https://docs.travis-ci.com/user/deployment/releases/

i'm currently experimenting with this on https://github.com/Kreyrock/Kreyrock based on https://github.com/dreibh/rsplib/blob/master/.travis.yml which is using it for test, but it can also be deployed.
